TW edits: Tutorial overview (#2)
keizobabybear/prefect: docs/tutorial/index.md

search:
  boost: 2
---
# Tutorial: Master workflow automations

This tutorial helps you get started with Prefect and become familiar with developing, deploying, and managing Prefect workflows.

> After you build your deployment, you can read other [guides](/guides) to learn how to integrate other services and add more features to your workflows.

Reviewer: ```suggestion
Once you've deployed your first workflow, you can read other [guides](/guides) to learn how to apply Prefect's features to more specific problems.
```

Reviewer: The main change I'm advocating for here is that we want to use the word "deploy" as a verb rather than a noun.

Reviewer 2: Maybe move the section on "After..." to the end of the tutorial, as that seems like a good time to direct folks to what they can do next.

## Before you begin

> Complete the following steps to start the tutorial:

Reviewer: ```suggestion
To get started:
```

1. Install [Python](https://www.python.org/downloads/).

> 1. Install Prefect:

Reviewer 2: ```suggestion
1. Install Prefect:
<div class="terminal">
```

Reviewer 2: Wrapping in a `div` tag for additional styling that we use with bash commands and removing the indent. Trying to keep the format consistent.

>     ```bash

Reviewer 2: ```suggestion
```bash
```

>     pip install -U prefect

Reviewer 2: ```suggestion
pip install -U prefect
```

>     ```

Reviewer 2: ```suggestion
```
</div>
```

>     ```

Reviewer 2: ```suggestion
```
```

>     For more detailed instructions, see [Installation](/getting-started/installation/).

Reviewer 2: ```suggestion
For more detailed instructions, see [Installation](/getting-started/installation/).
```

1. Connect to [Prefect Cloud](https://app.prefect.cloud). You can sign up for a forever free [Prefect Cloud account](/cloud/) or accept your organization's invite to join their Prefect Cloud account. Alternatively, you can use Prefect Cloud and self-host a [Prefect server instance](/host/). If you choose this option, start a local Prefect server instance by running the `prefect server start` command.

1. Create a new account or sign in at [https://app.prefect.cloud/](https://app.prefect.cloud/).

> 1. Authenticate to Prefect Cloud from your environment:

Reviewer: ```suggestion
1. Authenticate with your Prefect Cloud credentials from your environment:
```

Reviewer: "Authenticate to" reads oddly to me.
